src/main/resources/db/migration/V038__Add_indexes_to_process_events.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX process_events_container_zip_event_idx ON process_events (container, zipfilename, event);
```

Comment:
There already exists an index on container + zipfilename. If we need this one, the other one should be dropped.

Reply:
@lgonczar I think we still need the existing index for container and zipfilename. 
part of the reports query fetches the data on `container and zipfilename`

Reply:
This new index can still be used by PostgreSQL to search by container and zipfilename, as they're the first two columns in the index.

Reply:
ok, I'll update the script to drop the existing column